beautifulSets.py

```diff
@@ -0,0 +1,39 @@
+"""
+Approach: Recursion with BackTracking
+TC O(k) for k valid permutations we will do
```

Comment:
Kindly check the complexity .

Reply:
Updated. Please help correct if I am missing something still.